soweego/linker/feature_extraction.py

```diff
@@ -327,3 +328,123 @@ def intersection_percentage_size(pair):
             return count_intersect / count_total if count_total > 0 else np.nan
 
         return fillna(concatenated.apply(intersection_percentage_size), self.missing_value)
+
+
+class OccupationCompare(BaseCompareFeature):
```

Comment:
I suggest to use a more eloquent name: here we are comparing pairs of **sets with Wikidata QIDs about occupations**

Reply:
`OccupationQidSet` for instance

Reply:
Lists or sets? Not sure

Reply:
it's `set` :smile:  and yes you're right! name has been changed to `OccupationQidSetCompare` in 7e80b014a04a2cbd20a6faf6f466a117317eb055